internal/conf/config.go

```diff
@@ -40,6 +43,12 @@ func (oc *OperatorConfig) validate() error {
 	if oc.WorkingNamespace == "" {
 		return fmt.Errorf("WorkingNamespace value [%s] invalid", oc.WorkingNamespace)
 	}
+
+	// Ensure that CAIssuerName is set.
+	if oc.CAIssuerName == "" {
+		return fmt.Errorf("CAIssuerName value [%s] invalid", oc.CAIssuerName)
```

Comment:
There is no need to manually validate that; you can replace `default` with `required:"true"` in the struct tag for `CAIssuerName`.
And when we're at it, the same can be done with `WorkingNamespace`. That will remove the need for `validate`.

Reply:
I didn't know about it, thanks! Done